examples/aishell/s0/conf/train_conformer.yaml

```diff
@@ -41,7 +41,7 @@ raw_wav: true
 collate_conf:
     # waveform level config
     wav_distortion_conf:
-        wav_dither: 0 # x means x/32768.0 in torchaudio
+        wav_dither: 1.0
```

Comment:
change it to 0.0, since our verified best conformer recipe do not use dither now.